### ISSUE TITLE
no audio http? simple shoutcast stream?

### ISSUE THREAD

**Issue by Dj-jom2x** [luanpotter/audioplayers#375]

**Comment by MahdiPishguy** [comment]
@Dj-jom2x 

i have this problem too, and i don't know how we can catch and trace errors, audios couldn't play is some device

**Comment by Dj-jom2x** [comment]
yeah so hard to find a working one. thats the very basic one and yet its so hard to implement :3 on real android app its easy to make one but i wanted to try flutter..

**Comment by Mamong** [comment]
This plugin is based on MediaPlayer on android. So if you can not play a song via MediaPlayer, neither you can play it via this plugin.  Refer[MediaPlayer support media format](https://developer.android.com/guide/topics/media/media-formats)

**Comment by Dj-jom2x** [comment]
all shoutcast are mp3 or aac.. but ours is mp3 :( still no sound

**Comment by taekungkub** [comment]
i have same problam but it work on ios. Android not working for me

**Comment by Andrewngabriel** [comment]
This can be fixed by putting `android:usesCleartextTraffic="true"` within your manifest.xml file inside your Android directory. Example below
```
<?xml version="1.0" encoding="utf-8"?>
<manifest ...>
    <uses-permission android:name="android.permission.INTERNET" />
    <application
        ...
        android:usesCleartextTraffic="true"
        ...>
        ...
    </application>
</manifest>
```

**Comment by luanpotter** [comment]
closed in favour of #383
